build/swivel/Sig.sol

```diff
@@ -24,22 +25,17 @@ library Sig {
       revert S();
     }
 
-    if(c.v != 27 && c.v != 28) {
+    if (c.v != 27 && c.v != 28) {
       revert V();
     }
 
-    return ecrecover(h, c.v, c.r, c.s);
-  }
+    address recovered = ecrecover(h, c.v, c.r, c.s);
 
-  /// @param h Hashed data which was originally signed
-  /// @param sig Valid ECDSA signature
-  /// @dev splitAndRecover should only be used if it is known that the resulting 
-  /// verifying bit (V) will be 27 || 28. Otherwise use recover, possibly calling split first.
-  /// @return The recovered address
-  function splitAndRecover(bytes32 h, bytes memory sig) internal pure returns (address) {
```

Comment:
this whole method is deleted. it was a bit of legacy that got written and never used. nothing calls this.. the contracts just use `recover` with `split` being used by the Fakes and Unit tests.